src/Bicep.Core.IntegrationTests/RegistryTests.cs

```diff
@@ -30,6 +30,7 @@ namespace Bicep.Core.IntegrationTests
     public class RegistryTests
     {
         private static ServiceBuilder Services => new ServiceBuilder();
+        private static IServiceProvider EmptyServiceProvider = StrictMock.Of<IServiceProvider>().Object;
```

Comment:
Integration tests are failing due to strict mock usage: https://github.com/Azure/bicep/actions/runs/5061642034/jobs/9086219882. Might want to switch to a non-strict mock or create a factory with all services set up to a default test implementation.

Reply:
already done, thanks.

Reply:
cc0ddee3bce2306156f2b418847ef3fd451e37c2